REQUIRE

```diff
@@ -1,4 +1,5 @@
 julia 0.4
+Compat 0.19.0
```

Comment:
I thought this was using the indexing changes in 0.19 but I guess I was thinking of a different repo.